### PR TITLE
[New] Process Created with a Duplicated Token

### PR DESCRIPTION
**Pull request by Samirbous**
using `process.Ext.effective_parent.executable` to detect impersonation using token duplicates from windows native binaries to run common lolbins or recently dropped unsigned ones :

image

### CODE REVIEW

File: rules/windows/privilege_escalation_create_process_with_token_unpriv.toml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2023/10/02"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup, process.Ext.effective_parent.executable"
```
Comment (w0rk3r) [inline rules/windows/privilege_escalation_create_process_with_token_unpriv.toml:5]:
```suggestion
min_stack_comments = "New fields added: process.Ext.effective_parent.executable, process.Ext.effective_parent.name, process.Ext.relative_file_creation_time, process.Ext.relative_file_name_modify_time"
```

File: rules/windows/privilege_escalation_create_process_with_token_unpriv.toml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Identifies the creation of a process impersonating the token of another user logon session. Adversaries may create a new
+process with a different token to escalate privileges and bypass access controls.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Process Created with a Duplicated Token"
+references = [
+    "https://lengjibo.github.io/token/",
```
Comment (w0rk3r) [inline rules/windows/privilege_escalation_create_process_with_token_unpriv.toml:21]: Getting a 404 on this one

**Review by w0rk3r — APPROVED** [review]

File: rules/windows/privilege_escalation_create_process_with_token_unpriv.toml (hunk 2 above)
Comment (DefSecSentinel) [reply rules/windows/privilege_escalation_create_process_with_token_unpriv.toml:21]: Same,

**Review by DefSecSentinel — APPROVED** [review]
Remove reference otherwise LGTM

**Pull request merged**